app/controllers/books_controller.rb

```diff
@@ -0,0 +1,14 @@
+class BooksController < ApplicationController
+  respond_to :json
+  skip_before_action :verify_authenticity_token
+  before_action :authorize_request
+
+  def show
```

Comment:
what status is this method rendering? find if there is a way to have the response call explicit

Reply:
Also, research if with this gem you can receive the page number and limit (for kaminari) through the endpoint call